src/garage/replay_buffer/replay_buffer.py

```diff
@@ -12,173 +12,75 @@
 import abc
 from abc import abstractmethod
```

Comment:
please use only package-level imports for non-garage packages (i.e. use this as `@abc.abstractmethod`)